app/client/cypress/integration/national.spec.js

```diff
@@ -0,0 +1,188 @@
+
+    cy.get('a[href*="https://www.epa.gov/nutrientpollution"]')
+      .filter(':visible')
+      .should('have.attr', 'href', 'https://www.epa.gov/nutrientpollution');
+    cy.get('a[href*="https://www.epa.gov/nutrientpollution"]')
+      .filter(':visible')
+      .should('have.attr', 'target', '_blank');
+    cy.get('a[href*="https://www.epa.gov/nutrientpollution"]')
+      .filter(':visible')
+      .should('have.attr', 'rel', 'noopener noreferrer');
+  });
+
+  it('Clicking the “Learn more about what EPA is doing to reduce nutrient pollution” link opens a new tab.', () => {
+    // since Cypress doesn't support multiple tabs, we'll do the next best thing
+    // (https://docs.cypress.io/guides/references/trade-offs.html#Multiple-tabs)
+
+    cy.findByText(
+      /^Learn more about what EPA is doing to reduce nutrient pollution/,
+    ).should(
+      'have.attr',
+      'href',
+      'https://www.epa.gov/nutrient-policy-data/what-epa-doing-reduce-nutrient-pollution',
+    );
+
+    cy.findByText(
+      /^Learn more about what EPA is doing to reduce nutrient pollution/,
+    ).should('have.attr', 'target', '_blank');
+
+    cy.findByText(
+      /^Learn more about what EPA is doing to reduce nutrient pollution/,
+    ).should('have.attr', 'rel', 'noopener noreferrer');
+  });
+
+  it('Clicking on of the images in the “Learn more about water types” section opens a new tab.', () => {
+    // since Cypress doesn't support multiple tabs, we'll do the next best thing
+    // (https://docs.cypress.io/guides/references/trade-offs.html#Multiple-tabs)
+
+    cy.get(
+      'a[href*="https://www.epa.gov/national-aquatic-resource-surveys/national-rivers-and-streams-assessment-2008-2009-results"]',
+    ).should(
+      'have.attr',
+      'href',
+      'https://www.epa.gov/national-aquatic-resource-surveys/national-rivers-and-streams-assessment-2008-2009-results',
+    );
+    cy.get(
+      'a[href*="https://www.epa.gov/national-aquatic-resource-surveys/national-rivers-and-streams-assessment-2008-2009-results"]',
+    ).should('have.attr', 'target', '_blank');
+    cy.get(
+      'a[href*="https://www.epa.gov/national-aquatic-resource-surveys/national-rivers-and-streams-assessment-2008-2009-results"]',
+    ).should('have.attr', 'rel', 'noopener noreferrer');
+  });
+
+  it('Clicking the “Consumer Confidence Report (CCR)” link opens a new tab.', () => {
+    // since Cypress doesn't support multiple tabs, we'll do the next best thing
+    // (https://docs.cypress.io/guides/references/trade-offs.html#Multiple-tabs)
+
+    cy.get('a[href*="https://www.epa.gov/ccr"]').should(
+      'have.attr',
+      'href',
+      'https://www.epa.gov/ccr',
+    );
+    cy.get('a[href*="https://www.epa.gov/ccr"]').should(
+      'have.attr',
+      'target',
+      '_blank',
+    );
+    cy.get('a[href*="https://www.epa.gov/ccr"]').should(
+      'have.attr',
+      'rel',
+      'noopener noreferrer',
+    );
+  });
+});
```

Comment:
This test belongs in the other `describe` block (other tab). It also should follow the same things I mentioned above: don't use `cy.get()` and chain `should()`s.

Here's a simplified version of the block, but it still needs to be moved to the other `describe` block:
```suggestion
  it('Clicking the “Consumer Confidence Report (CCR)” link opens a new tab.', () => {
    // since Cypress doesn't support multiple tabs, we'll do the next best thing
    // (https://docs.cypress.io/guides/references/trade-offs.html#Multiple-tabs)
    cy.findByText('Consumer Confidence Report (CCR)')
      .should('have.attr', 'href', 'https://www.epa.gov/ccr')
      .should('have.attr', 'target', '_blank')
      .should('have.attr', 'rel', 'noopener noreferrer');
  });
});
```